sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -2123,7 +2124,7 @@ export interface GeoReplication {
 }
 
 // @public
-export type GeoReplicationStatusType = string;
+export type GeoReplicationStatusType = "live" | "bootstrap" | "unavailable";
```

Comment:
Minor: Breaking change, better have it in ChangeLog.

Reply:
It's to change back to the last GAed interface from a mistake in preview version.  There's no change for production customer. Better not mention it to avoid confusing.